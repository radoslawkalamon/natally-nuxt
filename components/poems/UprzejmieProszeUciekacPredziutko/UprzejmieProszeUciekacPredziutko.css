.uprzejmie-prosze-uciekac-predziutko__image {
  display: block;
  height: auto;
  margin: 0 auto;
  transform: rotate(135deg);
  width: 80px;
}
