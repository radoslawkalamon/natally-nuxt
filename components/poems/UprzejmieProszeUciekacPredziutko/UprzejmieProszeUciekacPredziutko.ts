import Vue from 'vue'

export default Vue.extend({
  name: 'PoemsUprzejmieProszeUciekacPredziutko'
})
