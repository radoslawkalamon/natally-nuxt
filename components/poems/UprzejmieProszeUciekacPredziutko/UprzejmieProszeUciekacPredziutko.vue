<template>
  <svg
    class="uprzejmie-prosze-uciekac-predziutko"
    data-test="poems-uprzejmie-prosze-uciekac-predziutko"
    viewBox="0 0 11 13"
    xmlns="http://www.w3.org/2000/svg"
  >
    <title>Czerwona b≈Çyskawica</title>
    <path d="M0.852 12.68C0.772 12.68 0.732 12.64 0.732 12.56C0.732 12.52 0.74 12.496 0.755999 12.488L4.92 7.95198L0.684 6.70398C0.636 6.68798 0.612 6.65198 0.612 6.59598C0.612 6.53998 0.628 6.50798 0.66 6.49998L10.128 0.715979C10.144 0.707979 10.164 0.703979 10.188 0.703979C10.212 0.703979 10.236 0.711979 10.26 0.727979C10.292 0.743979 10.308 0.77198 10.308 0.81198C10.308 0.84398 10.296 0.871979 10.272 0.895979L6.12 5.41998L10.368 6.48798C10.424 6.50398 10.452 6.53198 10.452 6.57198C10.452 6.62798 10.432 6.66798 10.392 6.69198L0.912 12.656C0.895999 12.672 0.875999 12.68 0.852 12.68Z" />
  </svg>
</template>

<script src="./UprzejmieProszeUciekacPredziutko.ts" lang="ts"></script>
<style src="./UprzejmieProszeUciekacPredziutko.css" scoped></style>
