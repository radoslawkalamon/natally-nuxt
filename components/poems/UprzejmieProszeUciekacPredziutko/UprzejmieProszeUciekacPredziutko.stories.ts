import { Meta, Story } from '@storybook/vue'
import UprzejmieProszeUciekacPredziutko from './UprzejmieProszeUciekacPredziutko.vue'

const meta: Meta = {
  title: 'Poems / Uprzejmie Proszę Uciekać Prędziutko',
  component: UprzejmieProszeUciekacPredziutko
}
export default meta

export const Default: Story = () => ({
  components: { UprzejmieProszeUciekacPredziutko },
  template: '<UprzejmieProszeUciekacPredziutko />'
})
