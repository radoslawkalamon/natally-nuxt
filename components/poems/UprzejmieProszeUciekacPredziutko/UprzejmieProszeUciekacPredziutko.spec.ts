import UprzejmieProszeUciekacPredziutko from '@/components/poems/UprzejmieProszeUciekacPredziutko/UprzejmieProszeUciekacPredziutko.vue'
import { shallPassUnitSanityTest } from '@/devtools/jest.common.spec'

describe('Poems / Uprzejmie Proszę Uciekać Prędziutko', () => {
  describe('Unit', () => {
    shallPassUnitSanityTest({ component: UprzejmieProszeUciekacPredziutko })
  })
})
