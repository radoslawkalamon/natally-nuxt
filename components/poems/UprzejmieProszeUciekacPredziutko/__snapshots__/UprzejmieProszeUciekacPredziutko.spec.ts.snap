// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Poems / Uprzejmie Proszę Uciekać Prędziutko Unit shall pass unit sanity test 1`] = `"<p data-test=\\"poems-uprzejmie-prosze-uciekac-predziutko\\" class=\\"uprzejmie-prosze-uciekac-predziutko\\"><img alt=\\"Czerwona błyskawica\\" src=\\"./images/czerwona-blyskawica.svg\\" class=\\"uprzejmie-prosze-uciekac-predziutko__image\\"></p>"`;
